### ISSUE TITLE
Ensure parsing errors

### ISSUE THREAD

**Issue by guybedford** [systemjs/systemjs#379]
Apparently Traceur parsing errors aren't being reported correctly in the browser. See https://github.com/jspm/jspm-cli/issues/596. They may have changed this error not to be an array.

**Comment by guybedford** [comment]
@flyingmutant I did a few tests and was unable to replicate your issue actually here, any tips welcome.

**Comment by flyingmutant** [comment]
Here is my test repo: https://github.com/flyingmutant/jspm-ng2-undefined-issue

Just `jspm install` and `python -m SimpleHTTPServer`. You'll see an error `Potentially unhandled rejection [2] TypeError: undefined is not a function`, which is caused by the fact that the imported objects are undefined for some reason.

**Comment by guybedford** [comment]
Ok right - well the good news is that in latest chrome this error comes up as `bootstrap is not a function`. I tried fixing the rtt thing and it didn't resolve that error so it is something else.

The other issue was actually something that is resolved in SystemJS 0.15, being posted into jspm very soon with the next release.

I managed to test that out too, but then it lead to another bug (for interest -

![screen shot 2015-03-20 at 16 43 58](https://cloud.githubusercontent.com/assets/598730/6753659/58e6de2a-cf20-11e4-97e4-f381218ace9b.png)

It's a tough integration you're working on, but really nice work. Thanks for the test case, unfortunately I can't do too much more on it right now, but let me know if you have specific questions.
